node_modules/gh-core/lib/db.js

```diff
@@ -253,6 +253,23 @@ var _setUpModel = function(sequelize) {
             'isGlobalAdmin': function() {
                 return false;
             },
+            'hide': function(ctx) {
```

Comment:
`hide` isn't really accurate. Something like `publicProfile` or `profile` might be better?